src/explore-education-statistics-common/src/components/form/rhf/RHFFormTextArea.tsx

```diff
@@ -0,0 +1,41 @@
+import FormGroup from '@common/components/form/FormGroup';
+import FormCharacterCount from '@common/components/form/FormCharacterCount';
+import FormBaseTextArea, {
+  FormTextAreaProps,
+} from '@common/components/form/FormBaseTextArea';
+import React from 'react';
+import { useWatch } from 'react-hook-form';
+
+export default function RHFFormTextArea({
+  id,
+  maxLength,
+  name,
+  ...props
+}: FormTextAreaProps) {
+  const value = useWatch({ name });
+
+  const hasMaxLengthError = maxLength
+    ? (value?.length ?? 0) > maxLength
+    : false;
```

Comment:
Similar to other comment, would be better to place this in the conditional to only run it if necessary

Reply:
It's removed now.